app/assets/javascripts/account/AccountService.js.coffee

```diff
@@ -92,6 +93,12 @@ AccountService = ($state, $auth, $modal, $http, ShortFormApplicationService) ->
       return
     )
 
+  Service.getMyApplications = ->
```

Comment:
is there a reason why you chose `myApplications` as opposed to `userApplications` or `currentUserApplications`?